admin/javascript/src/components/hidden-field/index.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import SilverStripeComponent from 'silverstripe-component';
+
+class HiddenFieldComponent extends SilverStripeComponent {
+
+  constructor(props) {
+    super(props);
+
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  render() {
+    return (
+      <div className="field hidden">
+        <input {...this.getInputProps()} />
+      </div>
+    );
+  }
+
+  getInputProps() {
+    return {
+      className: ['hidden', this.props.extraClass].join(' '),
+      id: this.props.name,
+      name: this.props.name,
+      onChange: this.props.onChange,
+      type: 'hidden',
+      value: this.props.value,
+    };
```

Comment:
This is probably a case where you should spread the props, so the user has a public api to the attributes..